packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -99,42 +94,24 @@ function sortByTimestampDescending(a: Transfer, b: Transfer) {
     : 1
 }
 
-const multiChainFetchList: ChainPair[] = [
-  {
-    parentChain: ChainId.Ethereum,
-    chain: ChainId.ArbitrumOne
-  },
-  {
-    parentChain: ChainId.Ethereum,
-    chain: ChainId.ArbitrumNova
-  },
-  // Testnet
-  {
-    parentChain: ChainId.Goerli,
-    chain: ChainId.ArbitrumGoerli
-  },
-  {
-    parentChain: ChainId.Sepolia,
-    chain: ChainId.ArbitrumSepolia
-  },
-  // Orbit
-  {
-    parentChain: ChainId.ArbitrumSepolia,
-    chain: ChainId.StylusTestnet
-  },
-  ...getOrbitChains().map(orbitChain => {
-    return {
-      parentChain: orbitChain.partnerChainID,
-      chain: orbitChain.chainID
-    }
-  }),
-  ...getCustomChainsFromLocalStorage().map(chain => {
-    return {
-      parentChain: chain.partnerChainID,
-      chain: chain.chainID
+function getMultiChainFetchList(): ChainPair[] {
+  return getChains().flatMap(chain => {
+    // We only grab deposit destinations because we don't want duplicates and we need the parent chain
+    const depositDestinationChainIds = chain.partnerChainIDs ?? []
```

Comment:
since arbitrum sdk 3.2.0 `partnerChainIDs` is mandatory so no need to default + let's rename
```suggestion
    const childChainIds = chain.partnerChainIDs
```